sirepo/template/radia.py

```diff
@@ -14,6 +14,7 @@
 from pykern import pkjinja
 from pykern.pkdebug import pkdc, pkdp
 from scipy.spatial.transform import Rotation
```

Comment:
use qualified import so have a chance at figuring out that it isn't local. `transform.Rotation` is better than `Rotation`, because you don't know if the class `Rotation` is defined locally or not.

Reply:
50f21c4